readme.md

```diff
@@ -54,7 +66,14 @@ console.log(results);
 
 Your Answer:
 ```text
-
+ You are missing the two params that it's looking for error and success.
+
+you could rewrite it something like:
+AuthorModel.find({name: 'Bob'}, function(err, name){
+  if (err)
+  console.log('i'm a failure)
+  else console.log('what's my name)
+  })
```

Comment:
:+1:

And, most notably, the callback function that accepts those params.  Which you included.  :)